lib/reek/context/module_context.rb

```diff
@@ -7,6 +9,19 @@ module Context
     # A context wrapper for any module found in a syntax tree.
     #
     class ModuleContext < CodeContext
+      include Enumerable
```

Comment:
I don't agree wholeheartedly but I can see why this might get confusing down the road. Changed.